packages/terra-worklist-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -410,15 +561,26 @@ function WorklistDataGrid(props) {
     const cellCoordinates = { row: focusedRow, col: focusedCol };
     let nextRow = cellCoordinates.row;
     let nextCol = cellCoordinates.col;
-
+    let activateMultiRowSelection = false;
     const key = event.keyCode;
     switch (key) {
       case KeyCode.KEY_UP:
+        if (event.shiftKey) {
+          activateMultiRowSelection = true;
+        }
         nextRow -= 1;
         break;
       case KeyCode.KEY_DOWN:
+        if (event.shiftKey) {
+          activateMultiRowSelection = true;
+        }
         nextRow += 1;
         break;
+      case KeyCode.KEY_SHIFT:
+        if (cellCoordinates.row !== 0 && !rowSelection?.current?.rangeUsingFocusedRow?.anchorRow) {
```

Comment:
How could rowSelection be null if it is a reference variable?  Do we need the "?" after it?

Reply:
Removed